src/utils/CalendarUtils/CalendarUtils.js

```diff
@@ -44,3 +45,9 @@ export const getCalendarDataByIndexes = (calendarData, weekIndex, dayIndex) => {
 
   return calendarData.children[0].children[weekIndex];
 };
+
+export const containerExists = selector => {
+  const element = $(selector);
+
+  return javaScriptUtils.isDefined(element);
+};
```

Comment:
I'm just curious, why are you put it here this function instead of `src/utils/JavaScriptUtils/JavaScriptUtils.js`?

Reply:
Good question. It was somewhat tough to decide to which utils-group the function belongs to. I'm of the opinion that `JavaScriptUtils` is composed of generic and pure JS functions without additional 3rd-party packages. However, it's by no means a strict rule which should be followed in all similar cases.

Reply:
Resolved and merged the PR. Feel free to come up with additional ideas in a separate issue/PR.